### PR TITLE
Apply formatting to "Utilities"

### PR DESCRIPTION
**Pull request by dsyme**
This is a big chunk of formatting

lib.fs failed formatting, I'm not sure why

### CODE REVIEW

File: src/Compiler/Service/ServiceLexing.fsi [hunk 1]
```diff
@@ -6,7 +6,6 @@ open System
 open System.Threading
 open FSharp.Compiler
 open FSharp.Compiler.Text
-
```
[the remark this answers is not in the document]
Comment (nojaf) [reply src/Compiler/Service/ServiceLexing.fsi:9]: @dsyme I noticed that one as well. This is fixed in https://github.com/fsprojects/fantomas/issues/2258 and will be part of the next alpha.

[the remark this answers is not in the document]
Comment (dsyme) [reply src/Compiler/Service/ServiceLexing.fsi:9]: Thanks!

**Comment by dsyme** [comment]
@vzarytovskii we should review each file, but on a first glance this looked good to go on.

File: src/Compiler/Utilities/TaggedCollections.fs [hunk 2]
```diff
@@ -987,95 +1193,123 @@ module MapTree =
...
+                    let c = m1.Comparer.Compare(kvp1.Key, kvp2.Key) in
+
+                    if c <> 0 then
+                        c
+                    else
+                        Unchecked.compare kvp1.Value kvp2.Value)
+                // Cast m2 to the exact same type as m1, see 4884.
+                // It is not OK to cast m2 to seq<KeyValuePair<'Key,'T>>, since different compares could permute the KVPs.
+                m1
+                (m2 :?> Map<'Key, 'T, 'ComparerTag>)
```
Comment (vzarytovskii) [inline src/Compiler/Utilities/TaggedCollections.fs:1299]: That looks a bit odd, `m1` on its own line.

**Review by vzarytovskii — APPROVED** [review]
Most of it looks fine to me, I think I only noticed one odd formatting.

File: src/Compiler/Utilities/TaggedCollections.fs (hunk 2 above)
Comment (nojaf) [reply src/Compiler/Utilities/TaggedCollections.fs:1299]: The function application `Seq.compareWith` takes three arguments and cannot be formatted on one line. See https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-application-expressions
Nothing odd here.

Comment (vzarytovskii) [reply src/Compiler/Utilities/TaggedCollections.fs:1299]: Oh, you're righ. Big compare function confused me a bit. We should probably bind it to something and use as named function, should be easier to read.

**Pull request merged**